helm-chart/binderhub/schema.yaml

```
@@ -551,3 +534,10 @@ properties:
       This is not a supported configuration yet by this Helm chart, but may be
       in the future. This schema entry was added because
       jupyterhub/mybinder.org-deploy configured a Networkpolicy resource here.
+
+  # Deprecated
+  cors:
+    type: object
+    additionalProperties: false
```

```suggestion
    additionalProperties: true
```

Without this being true, a user won't be able to get informed via our NOTES.txt, because helm will fail even before that if someone passed something nested under this.

---

@yuvipanda i dont think you tested the deprecation logic, this must allow unlisted properties as well i think!

Can you also set the additionalProperties to true explicitly on the other deprecated entry for consistency of having an explicit choice for all our objects?

---

Doing this, and you can test the deprecation logic

tools/generate-schema...
helm template bh helm-chart/binderhub --set cors.stuff=...

---

Resolved by my small schema fixes commit.